tests/integration/test_ec2.py

```diff
@@ -13,23 +13,52 @@
 from awscli.testutils import unittest, aws
 
 
-class BaseEC2Test(unittest.TestCase):
-    def assert_dry_run_success(self, command):
+class TestDescribeInstances(unittest.TestCase):
+    def setUp(self):
+        self.prefix = 'ec2 describe-instances --region us-west-2'
+
+    def test_describe_instances_with_id(self):
+        command = self.prefix + ' --instance-ids malformed-id'
         result = aws(command)
-        expected_response = ('Request would have succeeded, '
-                             'but DryRun flag is set.')
-        self.assertIn(expected_response, result.stderr)
+        self.assertIn('InvalidInstanceID.Malformed', result.stderr)
```

Comment:
Those tests are there to verify that the common cases for this operation are covered since we're modifying it. I'll see if I can find a way to reproduce the issue, but either way I think they should be kept.